CHANGELOG.md

```diff
@@ -4,14 +4,145 @@ All notable changes to this project will be documented in this file.
 
 This projects adheres to [Keep a CHANGELOG](http://keepachangelog.com/).
 
-The `major.minor` version numbers are based on the PHP version for which compatibility check support was added.
-The `patch` version numbers are specific to this library.
+Up to version 8.0.0, the `major.minor` version numbers were based on the PHP version for which compatibility check support was added, with `patch` version numbers being specific to this library.
+From version 8.0.0 onwards, [Semantic Versioning](http://semver.org/) is used.
 
 
 ## [Unreleased]
 
 _Nothing yet._
 
+## [8.0.0] - 2017-xx-xx
+
+**IMPORTANT**: This release contains a **breaking change**. Please read the below information carefully before upgrading!
+
+The directory layout of the PHPCompatibility standard has been changed for improved compatibility with Composer.
+This means that the PHPCompatibility standard no longer extends from the root directory of the repository, but now lives in its own subdirectory `/PHPCompatibility`.
+
+This release also bring compatibility with PHPCS 3.x to the PHPCompatibility standard.
+
+There are two things you will need to be aware of:
+* The path to the PHPCompatibility standard has changed.
+* If you intend to upgrade to PHPCS 3.x, the path to the `phpcs` script has changed (upstream change).
+
+Please follow the below upgrade instructions carefully. This should be a one-time only action.
+
+### Upgrade instructions
+
+### Before upgrading
+
+If you had previously made accommodations for the old directory layout, you should remove any such _"hacks"_ (meant in the kindest of ways) now.
+
+By this we mean: symlinks for the PHPCompatibility install to the `PHP_CodeSniffer/CodeSniffer/Standards` directory, scripts to move the sniffs files to the PHPCS directory, scripts which made symlinks etc.
+
+So, please remove those first.
+
+> **Side-note**:
+>
+> If you had previously forked this repository to solve this issue, please consider reverting your fork to the official version or removing it all together.
+
+### Upgrading: re-registering PHPCompatibility with PHP CodeSniffer
+
+External PHP CodeSniffer standards need to be registered with PHP CodeSniffer. You have probably done this the first time you used PHPCompatibility or have a script or Composer plugin in place to do this for you.
+
+As the directory layout of PHPCompatibility has changed, the path previously registered with PHP CodeSniffer will no longer work and running `phpcs -i` will **_not_** list PHPCompatibility as one of the registered standards.
+
+#### Using a Composer plugin
+
+If you use Composer, we recommend you use a Composer plugin to sort this out. In previous install instructions we recommended the SimplyAdmin plugin for this. This plugin has since been abandoned. We now recommend the DealerDirect plugin.
+```bash
+composer remove --dev simplyadmire/composer-plugins
+composer require --dev dealerdirect/phpcodesniffer-composer-installer
```

Comment:
I really would recommend `0.3` (known to be working) instead of the default (currently 0.4, which seems to have an issue).